docs/install/install-cave-beta-v20.md

```diff
@@ -123,7 +113,7 @@ Please note, running CAVE in a Virtual Machine does have reduced functionality t
 
 #### Download and Installation Instructions
 
-1. Download the zipped file containing the virtual machine: [**CentOS7-Unidata-CAVE-20.3.2-0.2** <i class="fa fa-download"></i>](https://downloads.unidata.ucar.edu/awips2/20.3.2/windows/CentOS7-Unidata-CAVE-20.3.2-0.2.zip)
+1. Download the zipped file containing the virtual machine: [**CentOS7-Unidata-CAVE-20.3.2-0.2** <i class="fa fa-download"></i>](https://downloads.unidata.ucar.edu/awips2/20.3.2/windows/CentOS7-Unidata-CAVE-20.3.2-0.3.zip)
```

Comment:
Line 116 - change text of link to have updated version
                 **CentOS7-Unidata-CAVE-20.3.2-0.3** 
Line 118 - change text of link to have updated version
                  Select the file called **"CentOS 7 - Unidata CAVE 20.3.2-0.3.vmx"**.